### DISCUSSION TITLE
The CTkLabel might miss something here?

### DISCUSSION THREAD

**Discussion by jvapal** [TomSchimansky/CustomTkinter#765] [Q&A]
CTkLabel messed up. 

Guess I missed something here, or higher versions needed for?

Thanks,

**Comment by TomSchimansky** [comment]
So do you use the newest version?

**Reply by jvapal** [reply]
The latest 5.0.1 requires many changes, like master=...
So I stay with V4.5.4, and it worked great before.

Recently I did change my OS to CentOS 7, wonder something mismatch.

Last good run I used tcl/tk 8.7 with PopOS.

Thanks,

**Comment by jvapal** [comment]
Switched ubuntu, it worked. Thanks,
